src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -40,6 +44,7 @@ def configure(conf)
             @mdsdLogger = Liboutmdsdrb::SocketLogger.new(djsonsocket, acktimeoutms,
                 resend_interval_ms, conn_retry_timeout_ms)
             @mdsdTagPatterns = mdsd_tag_regex_patterns
+            @configured_max_record_size = min(max_record_size, MDSD_MAX_RECORD_SIZE)
```

Comment:
>min(max_record_size, MDSD_MAX_RECORD_SIZE) [](start = 42, length = 42)

I don't think this work for Ruby. see [https://www.ruby-forum.com/topic/76399](https://www.ruby-forum.com/topic/76399). you can do [max_record_size, MDSD_MAX_RECORD_SIZE].min